play-media-native/screens/AthleteDetail.tsx

```diff
@@ -106,23 +98,17 @@ export const AthleteDetailScreen = ({ route, navigation }) => {
     theme.colors.gray.DEFAULT;
   const textColor = getTextColor(accentColor) || theme.colors.white.DEFAULT;
 
+  const handleEditInfo = (id: string, title: string) => {
+    navigation.navigate("EditAthleteDetails", {
+      id,
+      title,
+    });
+  };
+
   return (
     <Screen>
-      <ScrollView style={styles.screenPadding}>
+      <ScrollView style={styles.screenPadding} onScroll={calcScrollOffset}>
```

Comment:
I get a console warning
`You specified `onScroll` on a <ScrollView> but not `scrollEventThrottle`. You will only receive one event. Using `16` you get all the events but be aware that it may cause frame drops, use a bigger number if you don't need as much precision.`

Reply:
Not getting it on Android but should be fixed for iOS now.